src/Controller/PageController.php

```diff
@@ -157,11 +157,23 @@ public function page(string $slug, Request $request)
         $formErrors = null;
         $formData = $this->getFromData($request->request);
         $formCampaignCode = null;
+        $newsPropertie = null;
```

Comment:
Small nitpick, should this be 'newsProperties'? If it's a pain to change nevermind but just thought I'd point it out.